direct/tpm2/policy_test.go

```diff
@@ -314,42 +324,32 @@ func TestPolicyPCRUpdate(t *testing.T) {
 		},
 	}
 
-	pcrRead := PCRRead{
-		PCRSelectionIn: selection,
-	}
-
-	pcrReadRsp, err := pcrRead.Execute(thetpm)
-	if err != nil {
-		t.Fatalf("Failed to read PCRSelection")
-	}
-
-	var expectedVal []byte
-	for _, digest := range pcrReadRsp.PCRValues.Digests {
-		expectedVal = append(expectedVal, digest.Buffer...)
-	}
-	t.Logf("expectedVal=%x", expectedVal)
-
-	// Hash algorithm must match the one in PolicySession.
-	expectedDigest := sha1.Sum(expectedVal)
+	expectedDigest := getExpectedPCRDigest(t, thetpm, selection, tpm.AlgSHA1)
 	t.Logf("expectedDigest=%x", expectedDigest)
 
-	wrongCase := append(expectedVal, []byte("wrongCase")...)
-	wrongDigest := sha1.Sum(wrongCase)
+	wrongDigest := sha1.Sum(expectedDigest[:])
+	t.Logf("wrongDigest=%x", wrongDigest)
```

Comment:
nit: Probably don't need to log this

Reply:
This was helpful for my debugging, but yes I agree. Resolved.